CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
+### Fixed
+
+- Configuration is no longer ignored
```

Comment:
The changelog entry is wrong. It should be something like this:

```
## 0.1.4 - TBD

### Added

- Nothing.

### Changed

- Nothing.

### Deprecated

- Nothing.

### Removed

- Nothing.

### Fixed

- [#11](https://github.com/zendframework/zend-expressive-authorization-rbac/pull/11)
  fixes missing configuration factories key.
```

Reply:
Fixed